lib/litmus_paper/configuration_file.rb

```diff
@@ -45,12 +51,78 @@ def service(name, &block)
       @services[name.to_s] = service
     end
 
+    def yaml_service(value)
+      value.each do |name, config|
+        config = Util.symbolize_keys(config)
+        checks = []
+        dependencies = config.fetch(:dependencies, []).map do |dep|
```

Comment:
would breaking up the dependencies & checks evaluations into separate methods make the `yaml_service` method easier to read?

Reply:
Slightly, and I've split them.